x-pack/plugins/security_solution/public/hosts/components/common/host_risk_score.test.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+import { HostRiskSeverity } from '../../../../common/search_strategy';
+import { TestProviders } from '../../../common/mock';
+import { HostRiskScore } from './host_risk_score';
+
+describe('HostRiskScore', () => {
+  it('renders', () => {
+    const { container } = render(
+      <TestProviders>
+        <HostRiskScore severity={HostRiskSeverity.high} />
```

Comment:
Please consider adding some more tests for each severity to make sure they all rendered with expected colour and test.